ansible/deploy-pulp3.yml

```diff
@@ -4,6 +4,10 @@
   become: true
   vars:
     ansible_python_interpreter: "/usr/bin/python3"
+    rabbitmq_config:
+      - rabbit:
+        - tcp_listeners:
+          - '"0.0.0.0"': 5672
```

Comment:
How about "localhost" as the default, so it's not open to the world, and it gets us ipv6?

Reply:
FWIW I tested with "localhost" as the value, and the pulp processes seemed to connect fine.